### ISSUE TITLE
Eternal NPC confusion after a battle inside the refugee center

### ISSUE THREAD

**Issue by l29ah** [CleverRaven/Cataclysm-DDA#46126]
# Describe the bug

After clearing the back bay with the help of the local inhabitants, a lot of NPCs became agitated, moving around and shout at each other.

# Steps To Reproduce

Shoot some zombies at the refugee center or just load the attached game.

# Expected behavior
NPCs calming down after some time.

# Screenshots
![](https://tinystash.undef.im/il/JSQfCEaDjKJPwLbBFVTxwPEJDUytvE7KgP64YMiWWLv7VQoXRfZgdxWmBZLqPKakwsYPTiBMLk56yRgqdNceK8e)

# Versions and configuration
- OS: Linux
    - OS Version: <unknown>
- Game Version: 0.E-8244-g56e754589c-dirty [64-bit]
- Graphics Version: Curses
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions],
    Alternative Map Key [alt_map_key],
    No Rail Stations [No_Rail_Stations]
]

# Additional context

[2020.12.11.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/5718754/2020.12.11.zip)

**Comment by l29ah** [comment]
After few hours of walking around they do seem to stop, but now i can't move around the refugee center as they just stopped wherever they've been and clog the passages: ![](https://tinystash.undef.im/il/u1KKpkPaZWc1iHXZY9oXnmfVMCCHThmi8CxkdJPN1TLZDYwLLpV7sSfACpouRDqUi85Q4yAXGm6FognKUewbHLs)

**Comment by GGgatherer** [comment]
Yep, happened to me a few weeks ago too. Had to short-range teleport over some buffoon that was in the way.
